Radzen.Blazor/RadzenDialog.razor

```diff
@@ -8,11 +8,51 @@
     <DialogContainer Dialog=@dialog ShowMask=@(dialog==dialogs.LastOrDefault()) />
 }
 
+@if (isSideDialogOpen)
+{
+    <aside
+        class="@($"rz-dialog-side {sideDialogOptions.CssClass}")"
+        tabindex="@(isSideDialogOpen ? "0" : "-1")"
+        style="@($"{sideDialogOptions.Style}{(sideDialogOptions.Width != null ? " width: {sideDialogOptions.Witdth}" : "")}")"
```

Comment:
Style with optional set Width, which overrides the "hard coded" 400px width in the css